Worksheet-B/README.MD

```diff
@@ -0,0 +1,26 @@
+Pseudocode:
+```
+Generate 10 words which are 5 letters long
+Select one word at random as the answer, this word = chosenword
+Display the 10 words
+
+sequence starts here
```

Comment:
This is OK, but might be better structured as a commonly used programming structure e.g. a while loop